abilities-api.php

```diff
@@ -0,0 +1,40 @@
+<?php
+/**
+ * Plugin Name: Abilities API
+ * Plugin URI: https://github.com/WordPress/abilities-api
+ * Description: Provides a standardized API for registering and executing abilities in WordPress
+ * Version: 0.1.0
+ * Author: WordPress Contributors
+ * License: GPL v2 or later
+ * Text Domain: abilities-api
+ *
+ * @package Abilities_API
+ */
+
+// Prevent direct access
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+// TODO: https://github.com/WordPress/abilities-api/pull/4/ also contains this
```

Comment:
I know this is already getting added in another PR, but I'm adding a temporary version to make testing easier.

Reply:
That's fine. It's helpful for the time being. It will also allow us to land this PR sooner if the other one is not finalized.